src/app/sections/sections.component.ts

```diff
@@ -220,7 +244,7 @@ export class SectionsComponent implements OnInit {
     this.calculatedValues = {
       ltv: ((loan_amount * 1.0) / propertyValue).toFixed(2),
       propertyValue: (propertyValue * 1.0).toFixed(2),
-      maxLoanAmount: (maxLtvSelectedPercent * propertyValue * 1.0).toFixed(2),
+      maxLoanAmount: (maxLtvSelectedPercent * propertyValue * 1.0 / 100).toFixed(2),
```

Comment:
Added for - Total Rent/ Total Gross rent value is incorrect. I have selected 1 unit and as per that it should be 3000 for that unit (Market rent) but it is displaying some other value.